src/Model/Elastica/EavElasticaQuery.php

```diff
@@ -214,6 +215,13 @@ public function addQueries(BoolQuery $query)
                         continue;
                     }
                     $value = (float) $value;
+                } elseif ($attr instanceof AttributeMultiSelect) {
```

Comment:
This is ok. But I can't find filter or query for Select attribute. Could you add one